### PR TITLE
Notification dropdown should basically show unread notifications ever…

### PR DESCRIPTION
**Pull request by krunal-buddyboss**
…ywhere

### All Submissions:

* [x] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [x] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes https://github.com/buddyboss/buddyboss-theme/issues/1096

### How to test the changes in this Pull Request:
  

1. Go to any notification read page
2. Click on notification drop down
3. It is showing read notification in drop down
4. It should show unread notification on drop down everywhere

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

**Review by dipskakadiya — APPROVED** [review]
Look Good 👍

File: src/bp-notifications/bp-notifications-template.php [hunk 1]
```diff
@@ -185,10 +185,11 @@ function bp_get_notifications_read_permalink( $user_id = 0 ) {
  */
 function bp_has_notifications( $args = '' ) {
 
+	$notification_read_flag = bp_parse_args( $args );
```
Comment (mehulkaklotar) [inline src/bp-notifications/bp-notifications-template.php:188]:
```suggestion
	$args = bp_parse_args( $args );
```

File: src/bp-notifications/bp-notifications-template.php [hunk 2]
```diff
@@ -185,10 +185,11 @@ function bp_get_notifications_read_permalink( $user_id = 0 ) {
...
 	// Get the default is_new argument.
-	if ( bp_is_current_action( 'unread' ) ) {
+	if ( bp_is_current_action( 'unread' ) || isset( $notification_read_flag[ 'unread' ] ) ) {
```
Comment (mehulkaklotar) [inline src/bp-notifications/bp-notifications-template.php:190]:
```suggestion
	if ( bp_is_current_action( 'unread' ) || isset( $args[ 'unread' ] ) ) {
```

File: src/bp-notifications/bp-notifications-template.php [hunk 3]
```diff
@@ -185,10 +185,11 @@ function bp_get_notifications_read_permalink( $user_id = 0 ) {
...
 		$is_new = 1;
-	} elseif ( bp_is_current_action( 'read' ) ) {
+	} elseif ( bp_is_current_action( 'read' ) || isset( $notification_read_flag[ 'read' ] ) ) {
```
Comment (mehulkaklotar) [inline src/bp-notifications/bp-notifications-template.php:192]:
```suggestion
	} elseif ( bp_is_current_action( 'read' ) || isset( $args[ 'read' ] ) ) {
```

**Review by mehulkaklotar — CHANGES_REQUESTED** [review]

**Comment by mehulkaklotar** [comment]
@krunal-buddyboss Closing this because we don't need this change in Platform.

**Pull request closed without merging**